nfc/CHANGELOG.rst

```diff
@@ -9,6 +9,16 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+master
+******
+
+Modified
+========
+
+* A debug info is removed for the NFC T2T and T4T libraries.
+* Fixed duplicated initial Frame Waiting Time (FWT) value setting.
+
```

Comment:
Several changes are missing from the changelog. The last update of binaries was made from commit **9f47647bbe73f02e4a8303382f91a379c32ea34c** in the source repo. Also according to the commit message (I didn't make tests) the **00dd13ea183f6e5a05b59fb5cf8f76114d9c8457** version doesn't contain the latest fix (getting the FWI param).

Reply:
I added missing changes into the changelog also I switched to the correct library revision